modules/app/stores/uiFilters.ts

```diff
@@ -6,7 +6,7 @@ SPDX-License-Identifier: AGPL-3.0-or-later
 
 */
 
-import create from 'zustand';
+import { create } from 'zustand';
```

Comment:
Just wondering why this was necessary. Doesn't seem zustand version was updated.

Reply:
When running the portal it kept complaining that is deprecated and that we should import it like this now